system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -27,6 +27,8 @@ using namespace cryptohelper;
 
 #include "workunit.hpp"
 
+const uint64_t CLdapSecManager::s_implementedFeatures = (makeBitMask(SMF_NUMBER_OF_INDICES) - 1) & ~makeBitMask(SMF_RetrieveUserData);
+
```

Comment:
Some of these use cases for the mask routines aren't very intuitive for secmanger developers.  When I think what you really seem to be doing is makeSecFeatureMaskAllExcept(SMF_NUMBER_OF_INDICES, SMF_RetrieveUserData)?  Maybe add an explict function that makes this more intuitive?